src/Infrastructure/Persistence/ApplicationDbContext.cs

```diff
@@ -62,15 +98,20 @@ public override Task<int> SaveChangesAsync(CancellationToken cancellationToken =
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());
+            {
+                modelBuilder.HasPostgresEnum("audit", "history_activity_type", new[] { "CREATE", "UPDATE_BEFORE", "UPDATE_AFTER", "DELETE" })
+                    .HasPostgresExtension("adminpack");
```

Comment:
Why do you add the "adminpack" extension? The entity framework should not directly interact with the filesystem of the database server.

Reply:
I didn't. The scaffold did. I have to google now what it is to figure it out if it should stay